### PR TITLE
Update Get-CaseHoldRule.md

### PR DESCRIPTION
**Pull request by pjsfonseca69**
timeout issue when running only the command Get-CaseHoldRule without any parameter

### CODE REVIEW

**Comment by chrisda** [comment]
@pjsfonseca69, where does your command get the $org variable from? We should declare it or remove it.

File: exchange/exchange-ps/exchange/Get-CaseHoldRule.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ Get-CaseHoldRule [[-Identity] <ComplianceRuleIdParameter>]
 ```
 
 ## DESCRIPTION
+In large environments, running this cmdlet without any parametersresults in a timeout. As a workaround, you can run the following command:
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Get-CaseHoldRule.md:30]:
```suggestion
In large environments, running this cmdlet without any parameters results in a timeout. As a workaround, you can run the following command:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by pjsfonseca69** [comment]
> @pjsfonseca69, where does your command get the $org variable from? We should declare it or remove it.

What do you mean, we don't use any $org variable we just run Get-CaseHoldRule

**Pull request merged**